tests/sqllogictests/suites/ee/02_ee_aggregating_index/02_0000_agg_index_base

```diff
@@ -289,6 +289,9 @@ INSERT INTO t VALUES (1,1,4), (1,2,1), (1,-1,4), (2,2,5), (1,3,3)
 statement ok
 CREATE AGGREGATING INDEX testi AS select abs(b) as bs, sum(a) from t where c > 1 group by bs
 
+statement ok
+UPDATE t SET c = 3 where b = -1
```

Comment:
This statement will make no change, why add it?

Reply:
To ensure update statement can pass when agg index exists